### PR TITLE
release 1.0.1

### PR DESCRIPTION
**Pull request by adietish**

### CODE REVIEW

**Review by sbouchet — APPROVED** [review]
LGTM

File: src/main/resources/META-INF/plugin.xml [hunk 1]
```diff
@@ -10,6 +10,10 @@
     </description>
     <change-notes>
         <![CDATA[
+        <h3>1.0.1</h3>
+        <ul>
+            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/611">Display routes for openshift clusters (#610)</a></li>
```
Comment (mohitsuman) [inline src/main/resources/META-INF/plugin.xml:15]:
```suggestion
            <li><a href="https://github.com/redhat-developer/intellij-kubernetes/issues/611">Display routes for OpenShift clusters (#610)</a></li>
```

**Review by mohitsuman — APPROVED** [review]
LGTM

**Comment by adietish** [comment]
plugin version `1.0.1.241` was submitted to the marketplace for approval

**Pull request merged**